cluster/addons/kube-ui/kube-ui-endpoint.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
```

Comment:
The issue at the moment is that the /api/proxy URLs are using the endpoints rather that the service address.
We may be able to upstream a change to  use the service address as the URL of the proxy. So this is a temporary hack.